tutorials/Tutorial_1_Basics_of_CrypTen_Tensors.ipynb

```diff
@@ -43,11 +43,17 @@
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "tensor([1., 2., 3.])\n",
-      "tensor([4., 5., 6.])\n"
+     "output_type": "error",
```

Comment:
@knottb is this supposed to be here?

Reply:
No, I'm pretty sure I accidentally committed this jupyter notebook and gitignore

Reply:
Could you try to fix it @smsunarto?

Reply:
Yeap